lib/migration_data/config.rb

```diff
@@ -0,0 +1,6 @@
+module MigrationData
+  @config = Struct.new(:skip_data_on_test).new(false)
```

Comment:
Let's make it universal - theoretically, this option could be useful for `cucumber` environment, for example. Or some other environment that can be created by developers manually.